server/app.js

```diff
@@ -1,5 +1,5 @@
-var version = "0.2.3-beta";
-var release = 20181118;
+var version = "0.2.4-beta";
+var release = 20190327;
```

Comment:
The release var only needs to change if the DB structure is changed. So this one isn't required (in fact it breaks some of my logic on #174 lol.

I think we're pushing for the next release to be 0.3.0 we usually merge to the version branch then do a pull to master for the release.

Reply:
LOL. No problems. How do you want me to add this to the change log? Just under 0.2.3 or?